Assets/MRTK/Tests/PlayModeTests/ScrollViewTests.cs

```diff
@@ -0,0 +1,400 @@
+{
+    public class ScrollViewTests : BasePlayModeTests
+    {
+        #region Tests
+
+        /// <summary>
+        /// Tests if near interaction with a pressable button item is reset after the user engages in a scroll drag.
+        /// User should be able to interact with other buttons right after scroll engage is finished.
+        /// </summary>
+        [UnityTest]
+        public IEnumerator ScrollEngageResetsNearInteractionWithChildren()
+        {
+            // Setting up three pressable buttons as scroll items
+            GameObject scrollObject = new GameObject();
+
+            GameObject button1 = InstantiatePrefab(TestButtonUtilities.PressableHoloLens2PrefabPath);
+            button1.transform.parent = scrollObject.transform;
+
+            GameObject button2 = InstantiatePrefab(TestButtonUtilities.PressableHoloLens2PrefabPath);
+            button2.transform.parent = scrollObject.transform;
+
+            GameObject button3 = InstantiatePrefab(TestButtonUtilities.PressableHoloLens2PrefabPath);
+            button3.transform.parent = scrollObject.transform;
+
+            // Setting up a vertical 2x1 scroll view. The first two items are visible
+            var scrollView = scrollObject.AddComponent<ScrollingObjectCollection>();
+            scrollView.ScrollDirection = ScrollingObjectCollection.ScrollDirectionType.UpAndDown;
+            scrollView.CellWidth = button1.GetComponent<NearInteractionTouchable>().Bounds.x;
+            scrollView.CellHeight = button1.GetComponent<NearInteractionTouchable>().Bounds.y;
+            scrollView.Tiers = 1;
+            scrollView.ViewableArea = 2;
+            scrollView.UpdateCollection();
+            scrollView.TypeOfVelocity = ScrollingObjectCollection.VelocityType.NoVelocitySnapToItem;
+            scrollObject.transform.position = Vector3.forward;
+
+            TestUtilities.PlayspaceToOriginLookingForward();
+
+            PressableButton button1Component = button1.GetComponentInChildren<PressableButton>();
+            PressableButton button2Component = button2.GetComponentInChildren<PressableButton>();
+
+            Assert.IsNotNull(button1Component);
+            Assert.IsNotNull(button2Component);
+
+            bool button1TouchBegin = false;
+            button1Component.TouchBegin.AddListener(() =>
+            {
+                button1TouchBegin = true;
+            });
+
+            bool button1TouchEnd = false;
+            button1Component.TouchEnd.AddListener(() =>
+            {
+                button1TouchEnd = true;
+            });
+
+            bool button1PressBegin = false;
+            button1Component.ButtonPressed.AddListener(() =>
+            {
+                button1PressBegin = true;
+            });
+
+            bool button1PressCompleted = false;
+            button1Component.ButtonReleased.AddListener(() =>
+            {
+                button1PressCompleted = true;
+            });
+
+            bool button2TouchBegin = false;
```

Comment:
There's an event counter class somewhere in the test utilities, might help with having all this code here.